### PR TITLE
Use different image sizes for immersive header

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
Use appropriately sized images on the immersive header. Had to break the seal on the `Breakpoint` trait, not sure if there's a better way to do it.

### CODE REVIEW

**Comment by paperboyo** [comment]
Woohoo! 2000px sounds (looks!) much better than 1300px. Even though, I'm on 2560px ;) Thanks, @sammorrisdesign!

**Comment by paperboyo** [comment]
@sammorrisdesign Noticed something... On all breakpoints showcased images are requested at 20px less that they are shown at 100% zoom - have the text column got wider? This results in unnecessarily blurry images. Similar problem to recently fixed: https://github.com/guardian/frontend/issues/9482, https://github.com/guardian/frontend/issues/9278, https://github.com/guardian/frontend/issues/10467, https://github.com/guardian/frontend/issues/10493. Do you want me to open a separate issue?

**Comment by sammorrisdesign** [comment]
@paperboyo Yeah those are mid-article images, not this. So separate issue.

**Comment by paperboyo** [comment]
New issue opened: https://github.com/guardian/frontend/issues/10947.

File: common/app/layout/Breakpoint.scala [hunk 1]
```diff
@@ -2,7 +2,7 @@ package layout
 
 import scalaz.syntax.std.option._
 
-sealed trait Breakpoint {
```
Comment (sndrs) [inline common/app/layout/Breakpoint.scala]: i have no idea if this is a good thing or a bad thing

**Comment by sndrs** [comment]
great apart from comment - might want to get @jennysivapalan or @janua to check that.

:+1: apart form that

File: common/app/layout/Breakpoint.scala (hunk 1 above)
Comment (johnduffell) [inline common/app/layout/Breakpoint.scala]: what happens if you don't break it? I thought sealed just meant you couldn't define Breakpoint instances in any other file.

Comment (sammorrisdesign) [inline common/app/layout/Breakpoint.scala]: Yup you're right, I was being an idiot. Reverted it now.

Comment (johnduffell) [inline common/app/layout/Breakpoint.scala]: :stuck_out_tongue_winking_eye:

**Pull request merged**